python/lsst/daf/butler/core/_butlerUri/http.py

```diff
@@ -110,15 +110,36 @@ def getHttpSession() -> requests.Session:
                     "export this variable.")
 
     session.verify = ca_bundle
+    log.debug("Session configured and ready.")
+
+    return session
+
+
+def _defaultTimeout() -> float:
```

Comment:
Why isn't this a module constant? Why does it have to be a function? If you think the timeout might become more complex later on can you add a comment here explaining that for now it's a simple constant but in the future it might be dynamic?

Reply:
To be honest I don't see why it would ever become more complex than a constant, not sure why I went with a function in the first place. I will switch to a constant.